2025/realsense/extensions/preset-firstrun.sh

```diff
@@ -0,0 +1,52 @@
+function post_family_tweaks__preset_configs() {
+	display_alert "$BOARD" "preset configs for rootfs" "info"
+	# Set PRESET_NET_CHANGE_DEFAULTS to 1 to apply any network related settings below
+	echo "PRESET_NET_CHANGE_DEFAULTS=1" > "${SDCARD}"/root/.not_logged_in_yet
+
+	# Enable WiFi or Ethernet.
+	#      NB: If both are enabled, WiFi will take priority and Ethernet will be disabled.
+	echo "PRESET_NET_ETHERNET_ENABLED=1" >> "${SDCARD}"/root/.not_logged_in_yet
+	echo "PRESET_NET_WIFI_ENABLED=0" >> "${SDCARD}"/root/.not_logged_in_yet
+
+	#Enter your WiFi creds
+	#      SECURITY WARN: Your wifi keys will be stored in plaintext, no encryption.
+	echo "PRESET_NET_WIFI_SSID='🔧'" >> "${SDCARD}"/root/.not_logged_in_yet
+	echo "PRESET_NET_WIFI_KEY='millionmultiply'" >> "${SDCARD}"/root/.not_logged_in_yet
+
+	#      Country code to enable power ratings and channels for your country. eg: GB US DE | https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2
+	echo "PRESET_NET_WIFI_COUNTRYCODE='US'" >> "${SDCARD}"/root/.not_logged_in_yet
+
+	#If you want to use a static ip, set it here
+	echo "PRESET_NET_USE_STATIC=1" >> "${SDCARD}"/root/.not_logged_in_yet
+	echo "PRESET_NET_STATIC_IP='10.59.0.11'" >> "${SDCARD}"/root/.not_logged_in_yet
```

Comment:
The IP should be 10.59.40.11 to work on the robot since it's /24?

Or maybe 10.59.40.11/16 and then it will work on the robot and your router